src/main/java/de/cerus/jdasc/command/permissions/GuildApplicationCommandPermissions.java

```diff
@@ -0,0 +1,33 @@
+package de.cerus.jdasc.command.permissions;
+
+import java.util.List;
+
+public class GuildApplicationCommandPermissions {
+    private final long id;
+    private final long application_id;
+    private final long guild_id;
+    private final List<ApplicationCommandPermissions> permissions;
+
+    public GuildApplicationCommandPermissions(long id, long application_id, long guild_id, List<ApplicationCommandPermissions> permissions) {
```

Comment:
Same as above

Reply:
I was just following discord's naming scheme 🙄

Reply:
Yeah, but the Java code conventions state that variables and fields have to be named in lowerCamelCase :)